osu.Game/Overlays/Profile/Header/Components/GroupBadge.cs

```diff
@@ -35,19 +35,27 @@ public GroupBadge(APIUserGroup group)
             CornerRadius = 8;
 
             TooltipText = group.Name;
+
+            if (group.IsProbationary)
+            {
+                Alpha = 0.6f;
+            }
         }
 
         [BackgroundDependencyLoader]
         private void load(OverlayColourProvider? colourProvider, RulesetStore rulesets)
         {
             FillFlowContainer innerContainer;
 
+            var bgColor = colourProvider?.Background6 ?? Colour4.Black;
+
             AddRangeInternal(new Drawable[]
             {
                 new Box
                 {
                     RelativeSizeAxes = Axes.Both,
-                    Colour = colourProvider?.Background6 ?? Colour4.Black
+                    // Normal background color is 0.75 opacity, probationary doesn't have this cause they are a bit transparent already
+                    Colour = group.IsProbationary ? bgColor : bgColor.Opacity(0.75f),
```

Comment:
This reads very badly. Why aren't all cases applied via `Alpha`

Reply:
you mean setting `Colour = bgColor` (or leaving it how it was before) and below that have `Alpha = group.IsProbationary? 1 : 0.75f` on the box drawable?

Reply:
Yes. Basically right now you're applying alpha at two different levels in two different ways, and it's very hard to mentally parse.